DataStructure/src/main/list/ResizableArray.java

```diff
@@ -0,0 +1,167 @@
+package main.list;
+
+/**
+ * Represents a resizable array of integer elements.
+ */
+
+public class ResizableArray {
+    private int size;
```

Comment:
There is no need to use size variable. Arrays have size property in java.

Reply:
what size property, I don't get it ?
I took size as an instance variable because we need to increment it and also this is used to handle size = 0 edge case in functions.